Scoop/Controller/ProfileViewController.swift

```diff
@@ -20,8 +21,9 @@ class ProfileViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        view.backgroundColor = .systemGray5
+        view.backgroundColor = .gray
```

Comment:
should we change all the .systemGray5's to just be .gray?

Reply:
No this is a mistake, should he systemGray5!